charmdir.go

```diff
@@ -246,6 +248,32 @@ func (dir *CharmDir) LXDProfile() *LXDProfile {
 	return dir.lxdProfile
 }
 
+// Manifest returns the Manifest representing the manifest.yaml file
+// for the charm expanded in dir.
+func (dir *CharmDir) Manifest() *Manifest {
```

Comment:
Not sure why we don't embed the common elements and let methods like these get hoisted.

Reply:
done